test/source/mock/fes/customer-url-fes-endpoints.ts

```diff
@@ -80,7 +80,7 @@ export const getMockCustomerUrlFesEndpoints = (config: FesConfig | undefined): H
     '/api/v1/message/FES-MOCK-EXTERNAL-ID/gateway': async ({ body }, req) => {
       const port = parsePort(req);
       if (parseAuthority(req) === standardFesUrl(port) && req.method === 'POST') {
-        // test: `compose - user@standardsubdomainfes.localhost:8001 - PWD encrypted message with FES web portal`
+        // test: `compose - user@standardsubdomainfes.localhost:{port} - PWD encrypted message with FES web portal`
```

Comment:
let's just remove `:{port}` from tests names, so it'll be `compose - user@standardsubdomainfes.localhost - PWD encrypted message with FES web portal`, same for other tests

Reply:
Sounds good!